features/references/manage_working_patterns.feature

```diff
@@ -0,0 +1,34 @@
+# FeatureID  EpicID StoryID TestID
```

Comment:
I think that the premise of matching means that you need to verify the JOH working pattern, then go to the allocation page for a hearing that is scheduled and listed for a specific day - if the hearing is scheduled for a day / hour that matches the pattern then the JOH should be allocated, if not then you should not be able to allocate the JOH to that hearing

Reply:
Given I am on judicial office holder Page
    And   the judicial office holder works Monday to Friday
    And  the judicial office holder is Full Time
    When  I view a hearing that is on Monday within standard working hours
    Then  the judicial office holder can only be assigned to the matching hearing